### PR TITLE
check params[:skip_translate].nil?

### PR DESCRIPTION
**Pull request by graymd**
no jira

fixes no method error

### CODE REVIEW

File: app/controllers/alchemy/admin/elements_controller.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def update
 
           Rails.logger.error "[ALCHEMY_LOCALE] updating contents for element in elements_controller#update: #{@element}"
           update_translations(@element) if @element.page.language.language_code == 'en'
-          update_skip_translate(params[:skip_translate])
+          update_skip_translate(params[:skip_translate]) unless params[:skip_translate].nil?
```
Comment (jsqu99) [inline app/controllers/alchemy/admin/elements_controller.rb:81]:
```
update_skip_translate(params[:skip_translate]) if params[:skip_translate]
```

**Comment by graymd** [comment]
I still want params[:skip_translate] to pass false.

File: app/controllers/alchemy/admin/elements_controller.rb (hunk 1 above)
Comment (graymd) [inline app/controllers/alchemy/admin/elements_controller.rb:81]: I still want the method to execute if `params[:skip_translate] = false` so I used `.nil?`

**Comment by jccarte** [comment]
LGTM

**Comment by jsqu99** [comment]
LGTM

**Comment by graymd** [comment]
pointing to upstream

**Pull request merged**